.ci/update-archetypes.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-export ARCHETYPE_REPO_SLUG=gluonhq/client-maven-archetypes
-export XML_LOCATION=src/main/resources/META-INF/maven/archetype-metadata.xml
+ARCHETYPE_REPO_SLUG=gluonhq/client-maven-archetypes
```

Comment:
`export` is not required?

Reply:
We do not want to set env variables. We just want to use them in the script.